### DISCUSSION TITLE
Plugin build process

### DISCUSSION THREAD

**Discussion by amstiel** [samdark/intellij-visual-studio-code-dark-plus#47] [Q&A]
Tried to adjust Buttons/Git Log styles, but it's not that easy, as syntax highlighting. 
To check applied changes to UI, I need to build a plugin, But I'm not aware of build process. Is there gradle script or something? Can you share it, please?

**Comment by samdark** [comment]
@amstiel I use IntelliJ IDEA Community Edition.

![image](https://user-images.githubusercontent.com/47294/139812415-877939ab-5e48-4c4c-ba3b-b25ea50e5473.png)

**Comment by samdark** [comment]
I won't mind having a gradle script though.

**Comment by amstiel** [comment]
Thanks! I'll try IntelliJ IDEA Community Edition. But what about JDK version and other environment?

**Comment by samdark** [comment]
I'm using openjdk-15.0.2 Same one as bundled with IDE. Language level is set to SDK default (15).

**Comment by amstiel** [comment]
@samdark Thanks for info!
